spec/requests/api/visitor/can_respond_with_list_of_backyard_articles_spec.rb

```diff
@@ -0,0 +1,73 @@
+RSpec.describe 'GET /api/backyards', type: :request do
+  let!(:backyard_articles_denmark) { 2.times { create(:backyard_article, location: 'Denmark') } }
+  let(:vistor_location) do
+    file_fixture('visitor_location.json').read
+  end
+ 
+
+  describe 'Successfully' do
+    let!(:backyard_articles) { 3.times { create(:backyard_article) } }
+    before do
+      stub_request(:get, 'https://nominatim.openstreetmap.org/reverse?accept-language=en&addressdetails=1&format=json&lat=59.32021&lon=18.37827')
+      .with(
+        headers: {
+          'Accept' => '*/*',
```

Comment:
what does the * do?

Reply:
It's a wildcard, accepting everything

Reply:
sweet! thanks

Reply:
No idea, this is part of the headers that we add to stub out the calls. It's from Webmock.